tests/unit/test_ocsort_parity.py

```
@@ -0,0 +1,236 @@
+        ("lowscore0_byte", BYTE, seq_lowscore(0)),
+        ("lowscore1_byte", BYTE, seq_lowscore(1)),
+    ]
+    return seqs
+
+
+# --------------------------------------------------------------------------
+# Comparison helpers
+# --------------------------------------------------------------------------
+
+
+def _by_id(rows):
```

Ruff (0.15.15)

[warning] Missing return type annotation for private function `_by_id`

(ANN202)